packages/arb-token-bridge-ui/src/components/TransferPanel/TokenDepositCheckDialog.tsx

```diff
@@ -110,8 +57,35 @@ export function TokenDepositCheckDialog(props: TokenDepositCheckDialogProps) {
   }, [type, symbol])
 
   return (
-    <Dialog {...props} title={title}>
-      <p className="pt-4">{textContent}</p>
+    <Dialog {...props} title={title} className="flex flex-col gap-4">
```

Comment:
Figma says bottom margin is 30px but it's not consistent with other modals, so let's stick with 20px.